src/main/java/org/gridsuite/study/server/service/NotificationService.java

```diff
@@ -111,6 +112,17 @@ public void emitStudyChanged(UUID studyUuid, UUID nodeUuid, String updateType) {
                 .build());
     }
 
+    @PostCompletion
```

Comment:
It's used everywhere, I think @niconoir told me to use it, maybe we can ask him ?

Reply:
@PostCompletion is used to ensure the method will be executed as soon as the transaction we are in ends. It might be useful to use it here to ensure error will be notified after possible database modifications are flushed or rolled back (the database at this point will be consistent).